src/pages/index.tsx

```diff
@@ -36,7 +36,7 @@ function HomepageHeader() {
               flexDirection: "row",
             }}
           >
-            <SVGImage Svg={chefHat.default} classNameOverride="chefHatSvg" />
+            <img src={chefHat.default} alt="chefHatSvg" />
```

Comment:
The `alt` attribute here isn't the same as `className` that was maybe what this was supposed to be changed to?

Or if it's meant to be alt, let's provide a description rather than a var name

Reply:
``` suggestion
            <img src={chefHat.default} alt="Chef's Hat" />
```